recipes/datasets/plugins/plugin_consensus.py

```diff
@@ -0,0 +1,66 @@
+from kili.plugins import PluginCore
+from kili.types import Label
+
+
+class PluginHandler(PluginCore):
+    """
+    Custom plugin instance
+    """
+
+    def aggregate_object_detection_label(self, object_job_key, labels_array):
+
+        json_response_array = {}
+
+        for key in object_job_key:
+            l_key = []
+            for _, label in enumerate(labels_array):
+                for annotation in label["jsonResponse"][key]["annotations"]:
+                    l_key.append(annotation)
+            json_response_array.update({key: {"annotations": l_key}})
+
+        return json_response_array
+
+    def on_submit(self, label: Label, asset_id: str) -> None:
+        """
+        Dedicated handler for Submit action
+        """
+        project_id = self.project_id
+
+        project_info = list(
+            self.kili.projects(project_id=project_id, fields=["minConsensusSize", "jsonInterface"])
+        )[0]
+
+        json_interface = project_info["jsonInterface"]
+        min_consensus_size = project_info["minConsensusSize"]
+
+        object_job_key = [
+            k for k, v in json_interface["jobs"].items() if v["mlTask"] == "OBJECT_DETECTION"
+        ]
+
+        labels_array = self.kili.labels(
+            project_id=project_id,
+            asset_id=asset_id,
+            type_in=["DEFAULT"],
+            fields=[
+                "author.email",
+                "author.id",
+                "id",
+                "jsonResponse",
+                "labelType",
+                "secondsToLabel",
+                "isLatestLabelForUser",
```

Comment:
If you only need `isLatestLabelForUser` and `jsonResponse` in the plugin, there is no need to fetch extra fields (you can probably keep `id`, but fetching info on `author` makes an unnecessary join in the DB